mne/event.py

```diff
@@ -827,3 +828,289 @@ def concatenate_events(events, first_samps, last_samps):
         events_out = np.concatenate((events_out, e2), axis=0)
 
     return events_out
+
+
+class Elekta_event(object):
```

Comment:
Can think about it. I would prefer classes, since there is basically no cost and they are easier to expand later. Also if the user wants to create his own categories/events, sanity checking is much more easier inside a class. (I would like to make some API expansions so that one can easily change averaging parameters or create new events later)